locals.tf

```
@@ -199,9 +199,9 @@ locals {
   # The name and description of the role and policy that allows read-only
   # access to the VNC-related SSM Parameter Store parameters in the
   # Images account.
```

Update comment to reflect addition of RDP:
```suggestion
  # access to the VNC-related and RDP-related SSM Parameter Store
  # parameters in the Images account.
```

This change should be batched with similar changes below.